src/pkg/bundler/localbundle.go

```diff
@@ -107,17 +110,37 @@ func (lo *LocalBundle) create(signature []byte) error {
 
 	message.HeaderInfof("🚧 Building Bundle")
 
-	// push uds-bundle.yaml to OCI store
-	bundleYAMLDesc, err := pushBundleYAMLToStore(store, bundle)
-	if err != nil {
-		return err
+	var bundleManifestDesc ocispec.Descriptor
+	if lo.isTofu {
+		bundleManifestDesc, err = pushBundleTFToStore(store, filepath.Join(lo.sourceDir, config.BundleTF))
+		if err != nil {
+			return err
+		}
+	} else {
+		// push uds-bundle.yaml to OCI store
+		bundleManifestDesc, err = pushBundleYAMLToStore(store, bundle)
+		if err != nil {
+			return err
+		}
 	}
```

Comment:
nit: style suggestion

```suggestion
	var bundleManifestDesc ocispec.Descriptor
	if lo.isTofu {
		// push uds-bundle.tf to OCI store
		bundleManifestDesc, err = pushBundleTFToStore(store, filepath.Join(lo.sourceDir, config.BundleTF))
	} else {
		// push uds-bundle.yaml to OCI store
		bundleManifestDesc, err = pushBundleYAMLToStore(store, bundle)
	}
	if err != nil {
		return err
	}
```